L/libclangex/build_tarballs.jl

```diff
@@ -62,16 +70,35 @@ for llvm_version in llvm_versions, llvm_assertions in (false, true)
         # so loading the library will always fail. We fix this in ClangCompiler.jl
         LibraryProduct("libclangex", :libclangex, dont_dlopen=true),
     ]
-    
+
+    # These are the platforms we will build for by default, unless further
+    # platforms are passed in on the command line
+    platforms = expand_cxxstring_abis(supported_platforms())
+    # disable riscv64
+    filter!(p -> arch(p) != "riscv64", platforms)
+    # disable aarch64 freebsd
+    filter!(p -> !(Sys.isfreebsd(p) && arch(p) == "aarch64"), platforms)
+
+    if llvm_version >= v"15"
```

Comment:
Is there ever a plan to build this for anything below LLVM 15 again? The LLVM versions listed above is at 18, so if we won't ever build it with those again, it is cleaner to just remove the conditional.